app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name" translate="false">AntennaPod</string>
+    <string name="noMediaMsgText">Item does not contain a media file</string>
```

Comment:
Please specify the string in `core/src/main/res/values/strings.xml`. Only that file is uploaded to translators.

Reply:
I don't want to just throw a new value in anywhere, and looking through the file, I'm unsure where it fits best. 
-> actions on feed items? empty list items? other? 

forgive my ignorance, yesterday was the first I've looked at this code and this is my first pull request

Reply:
I think it could fit next to `error_file_not_found`. It's not really a media player error message but there it is at least next to something similar.

> this is my first pull request

Welcome to AntennaPod :)

Reply:
Thank you!